Gateway/Request/CaptureRequest.php

```diff
@@ -40,18 +37,20 @@ public function build(array $buildSubject)
         if (!$payment instanceof OrderPaymentInterface) {
             throw new \LogicException('Order payment should be provided.');
         }
-
+        $cc_exp_month = $payment->getAdditionalInformation("cc_exp_month");
+        $cc_exp_year = $payment->getAdditionalInformation("cc_exp_year");
         if ($payment->getLastTransId() == '') {
             return [
                 'xCommand' => 'cc:sale',
                 'xAmount'   => $amount,
-                'xExp' => sprintf('%02d%02d', $payment->getAdditionalInformation("cc_exp_month"), substr($payment->getAdditionalInformation("cc_exp_year"), -2)),
+                'xExp' => sprintf('%02d%02d', $cc_exp_month, substr($cc_exp_year, -2)),
                 'xCVV' => $payment->getAdditionalInformation("xCVV"),
                 'xInvoice' => $order->getOrderIncrementId(),
                 'xCurrency' => $order->getCurrencyCode(),
                 'xCardNum' => $payment->getAdditionalInformation("xCardNum"),
                 'xIgnoreInvoice' => true,
-                'xTimeoutSeconds' => 55
+                'xTimeoutSeconds' => 55,
+                'xAllowDuplicate' => true
```

Comment:
Same for this

Reply:
It throws error when order place with same grand total with same time.
**Error**: Duplicate Transaction in payment.log